x-pack/plugins/security_solution/public/detections/components/rules/rule_actions_overflow/index.tsx

```diff
@@ -83,10 +85,18 @@ const RuleActionsOverflowComponent = ({
               onClick={async () => {
                 startTransaction({ name: SINGLE_RULE_ACTIONS.DUPLICATE });
                 closePopover();
+                const duplicateExceptions = await showBulkDuplicateExceptionsConfirmation();
+                if (!duplicateExceptions) {
+                  return;
+                }
```

Comment:
Why do we skip the entire bulk action if `duplicateExceptions` is falsy? Shouldn't we proceed without duplicating exceptions?